VogtPayroll6/Display.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll6
+{
+    class Display
+    {
+        public void PrintMenu(Employee[] empArray)
```

Comment:
I think you should pass in the Payroll here from `Program.cs`. That would be more logical. The display menu shouldn't create the payroll system. It should be passed the payroll system that it needs to work with.

This is called Inversion of Control via Dependency Injection. (one is the concept, and the other is a pattern to adhere to the concept)